app/lib/search_facet_populator.rb

```diff
@@ -17,25 +19,25 @@ def self.populate_from_schema
   # creates/updates a facet from a name, and returns the new SearchFacet.
   # To manually populate a new Alexandria convention facet from the rails console, run e.g.
   # SearchFacetPopulator.populate_facet_by_name('vaccination__route')
-  def self.populate_facet_by_name(facet_name, schema_object=nil)
+  def self.populate_facet_by_name(facet_identifier, schema_object=nil)
```

Comment:
Could `facet_id` work here, for less verbosity?

Reply:
since `identifier` is the name of the DB column it's stored in, I like using identifier here so it's not confused with the DB object ID.  Although, looking at this, it now doesn't make sense to have this method be called 'populate_facet_by_name'. :).  I'll update

Reply:
Makes sense!